account_tax_balance/tests/test_account_tax_balance.py

```diff
@@ -183,13 +184,14 @@ def test_tax_balance(self):
                 'name': 'Bank Fees',
                 'tax_ids': [(4, tax.id)]
             }), (0, 0, {
-                'account_id': tax.account_id.id,
+                'account_id': tax_account_id,
                 'debit': 0,
                 'credit': 10,
                 'name': 'Bank Fees',
-                'tax_line_id': tax.id,
+                'tax_repartition_line_id': tax_repartition_line.id,
```

Comment:
I don't know why you change this line. I think it should be as before. 🤔

Reply:
Because [ `tax_line_id` is a compute field](https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L2454-L2455) and doesn't have any method for the inverse.

Reply:
:+1: